rosco-manifests/src/test/groovy/com/netflix/spinnaker/rosco/manifests/kustomize/KustomizationFileReaderSpec.groovy

```diff
@@ -47,18 +58,20 @@ class KustomizationFileReaderSpec extends Specification {
         Kustomization k = kustomizationFileReader.getKustomization(baseArtifact,"kustomization.yml")
 
         then:
-        1 * clouddriverService.fetchArtifact(_ as Artifact) >> { Artifact a ->
-            return new Response("test", 200, "", [], new TypedString(kustomizationYaml.stripMargin()))
-        }
         k.getResources().sort() == ["deployment.yml", "service.yml"].sort()
         k.getSelfReference() == "https://api.github.com/repos/org/repo/contents/base/kustomization.yml"
     }
 
     def "getKustomization throws an exception if it can't find a valid kustomization file"() {
         given:
-        def clouddriverService = Mock(ClouddriverService) {
+        def mockCall = Mock(Call.class) {
+            execute() >> {
+                 return Response.error(404, ResponseBody.create(null, ""))
```

Comment:
Why 404 here when the previous mock returned 500?

Reply:
No specific reason, I will revert it back